### PR TITLE
[5.0] public Route Validators Access.

### PR DESCRIPTION
**Pull request by findstar**
add setValidators method .

to solve #7957 issue .

### CODE REVIEW

File: src/Illuminate/Routing/Route.php [hunk 1]
```diff
@@ -636,6 +636,19 @@ public static function getValidators()
 		);
 	}
 
+    /**
```
Comment (GrahamCampbell) [inline src/Illuminate/Routing/Route.php]: You must indent with tabs.

File: src/Illuminate/Routing/Route.php [hunk 2]
```diff
@@ -636,6 +636,19 @@ public static function getValidators()
...
+     * Set the route validators.
+     *
+     * @param array  $validators
```
Comment (GrahamCampbell) [inline src/Illuminate/Routing/Route.php]: Please put exactly 2 spaces before and after the word `array`.

**Comment by GrahamCampbell** [comment]
Why not just make the property public>

**Comment by findstar** [comment]
Ok. I just change protected property to public.

**Pull request merged**